samwise.js

```diff
@@ -7,31 +7,30 @@
 /* global clearTimeout, setTimeout, document */
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?